build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -61,34 +53,45 @@ steps:
       git merge $(node -p "require('./package.json').distro")
     displayName: Merge distro
 
-  - task: 1ESLighthouseEng.PipelineArtifactCaching.RestoreCacheV1.RestoreCache@1
+  - script: |
+      mkdir -p .build
+      node build/azure-pipelines/common/computeNodeModulesCacheKey.js > .build/yarnlockhash
+    displayName: Prepare yarn cache flags
```

Comment:
flags? This isn't really a flag. `Prepare yarn cache hash` or `Prepare yarn cache key` seems more appropriate

Reply:
Using the task name from vscode

Reply:
Eh, since this is our file then I would still suggest updating them since - to me at least - these don't really give a good indication what they're actually doing.